src/components/UserSelector.tsx

```diff
@@ -20,21 +38,14 @@
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">
-            Leanne Graham
-          </a>
-          <a href="#user-2" className="dropdown-item is-active">
-            Ervin Howell
-          </a>
-          <a href="#user-3" className="dropdown-item">
-            Clementine Bauch
-          </a>
-          <a href="#user-4" className="dropdown-item">
-            Patricia Lebsack
-          </a>
-          <a href="#user-5" className="dropdown-item">
-            Chelsey Dietrich
-          </a>
+          {users.map(user => (
+            <OneUser
+              user={user}
+              key={user.id}
```

Comment:
Good job using `user.id` as the key for the `OneUser` component. This ensures that each component instance is uniquely identified, which is important for React's reconciliation process.